app/services/schools/home.rb

```diff
@@ -12,5 +12,11 @@ def ects_with_mentors
         .eager_load(:teacher, :school, mentors: :teacher)
         .merge(MentorshipPeriod.ongoing)
     end
+
+    def mentors_with_ects
```

Comment:
Why is this `mentors_with_ects` when that isn't the query - is this just a list of mentors at the school?

Reply:
My understanding was that the cards we are iterating over will need their ECTs (mentees) included, but that is in the next ticket and out of scope right now. I was setting out the intention.

Reply:
OK, not all mentors will have ECTs all the time, is the intention not to show those do you know?

Reply:
There should be tickets to consider what to do if a mentor has no mentees or has more than could comfortably be shown. The initial design used an unordered list but I was imagining another status indicator too.